Content/Resources/Materials/FacePicker.mat

```diff
@@ -26,6 +26,6 @@ Material:
     m_Ints: []
     m_Floats: []
     m_Colors:
-    - _Tint: {r: 1, g: 1, b: 1, a: 1}
+    - _Tint: {r: 0, g: 0, b: 0, a: 1}
```

Comment:
I think you added the changes in `Content/Resources/Materials` by accident.

Reply:
```suggestion
    - _Tint: {r: 1, g: 1, b: 1, a: 1}
```